docs/src/fts.md

```diff
@@ -64,10 +65,23 @@ table.create_fts_index(["text1", "text2"])
 
 Note that the search API call does not change - you can search over all indexed columns at once.
 
+## Filtering
+
+Currently the LanceDB full text search feature supports *post-filtering*, meaning filters are
+applied on top of the full text search results. This can be invoked via the familiar
+`where` syntax:
+
+```python
+table.search("puppy").where("meta='foo'").limit(10).to_list()
```

Comment:
I know it technically doesn't make a difference, but might be clearer to put the limit before the where to hint at the post-filtering.

```suggestion
table.search("puppy").limit(10).where("meta='foo'").to_list()
```